libid/complex.cpp

```diff
@@ -0,0 +1,536 @@
+// SPDX-License-Identifier: GPL-3.0-only
+//
+// Complex.cpp: interface for the Complex class.
+//
+//////////////////////////////////////////////////////////////////////
+#include "Complex.h"
+
+//Complex::Complex(void)
+//    {
+//    }
+
+Complex::~Complex(void)
+    {
+    }
+
+Complex Complex::operator=(const double & value)	// Assignment to double Operator
+    {
+    x = value;
+    y = 0.0;
+    return *this;
+    }
+
+Complex Complex::operator=(const Complex & Cmplx1)	// Assignment Operator
```

Comment:
I prefer to name the argument `rhs` for "right-hand side", because it is the argument to the right of the assignment operator.`

This should return `Complex &` not `Complex`